kytos/core/interface.py

```diff
@@ -344,10 +347,15 @@ def _use_tags(
                     self.available_tags[tag_type] = available_copy
                     conflict = range_difference(tags, available_copy)
                     raise KytosTagsAreNotAvailable(conflict, self._id)
-        else:
+        elif isinstance(tags, list):
             result = self._remove_tags(tags, tag_type)
             if result is False:
                 raise KytosTagsAreNotAvailable([tags], self._id)
+        else:
+            try:
+                self.special_available_tags[tag_type].remove(tags)
```

Comment:
An invalid `tag_type` would raise a `KeyError` here too, let's handle it.

Reply:
Added statement to check `tag_type`